src/pkg/packager/lint/validator.go

```diff
@@ -6,98 +6,146 @@ package lint
 
 import (
 	"fmt"
+	"path/filepath"
 
 	"github.com/defenseunicorns/zarf/src/pkg/message"
-	"github.com/defenseunicorns/zarf/src/pkg/utils"
+	"github.com/defenseunicorns/zarf/src/pkg/utils/helpers"
 	"github.com/defenseunicorns/zarf/src/types"
 	"github.com/fatih/color"
 )
 
+type validationType int
+
+const (
+	validationError   validationType = 1
+	validationWarning validationType = 2
+)
+
 type validatorMessage struct {
-	yqPath      string
-	filePath    string
-	description string
-	item        string
+	yqPath         string
+	description    string
+	item           string
+	packageRelPath string
+	packageName    string
+	validationType validationType
```

Comment:
what about `type validation int`

so you can do

```go
type validatorMessage struct {
   ...
   Type validation
 }
 ```
 
 @Racer159 what's your take on stuttering w/ `type xyzType`

Reply:
After talking with @Racer159 we ended up changing it to category